### ISSUE TITLE
Vertical slider left labels and rules are mispositioned

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3489]
This file renders its labels and rules at the wrong locations. Fails in preview also.

```
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<title>Untitled</title>
<script type="text/javascript" src="alib/dojo/dojo/dojo.js" data-dojo-config="'parseOnLoad':true,'async':true,'packages':[{'name':'maqettaSamples','location':'../../../samples'},{'name':'maqetta','location':'../../../lib/maqetta'},{'name':'zazl','location':'../../../lib/zazl'},{'name':'gridx','location':'../gridx'},{'name':'clipart','location':'../../clipart'},{'name':'shapes','location':'../../shapes'},{'name':'widgets','location':'../../../lib/custom'}],'themeMap':[['Android',null,['themes/custom2/custom2.css']],['BlackBerry',null,['themes/custom2/custom2.css']],['iPad',null,['themes/custom2/custom2.css']],['iPhone',null,['themes/custom2/custom2.css']],['.*',null,['themes/custom2/custom2.css']]],'mblThemeFiles':[], mblLoadCompatPattern: /\/project2\/WebContent\/themes\/.*\.css$/"></script>
<script type="text/javascript">
require([
  "dijit/dijit",
  "dojo/parser",
  "maqetta/space",
  "maqetta/AppStates",
  "dijit/form/VerticalSlider",
  "dijit/form/VerticalRule",
  "dijit/form/VerticalRuleLabels",
  "dijit/form/HorizontalSlider",
  "dijit/form/HorizontalRule",
  "dijit/form/HorizontalRuleLabels"
]);
</script>
<style>@import "themes/claro/document.css";@import "themes/claro/claro.css";@import "app.css";
</style>
<script type="text/javascript" src="app.js"></script>
</head>
<body class="claro" data-maq-flow-layout="true" data-maq-ws="collapse" id="myapp" data-maq-appstates="{}">
 <div></div>
 <div data-dojo-type="dijit.form.VerticalSlider" intermediateChanges="false" discreteValues="Infinity" style="width: auto; height: 200px;">
   <ol data-dojo-type="dijit.form.VerticalRuleLabels" container="leftDecoration" style="width:3em;"></ol>
   <div data-dojo-type="dijit.form.VerticalRule" container="leftDecoration" style="width:5px;"></div>
 </div>
</body>
</html>
```

**Comment by JonFerraiolo** [comment]
Here is a snapshot:

![Vertical slider layout issue](http://i.imgur.com/D9jv2.png)

**Comment by JonFerraiolo** [comment]
I see this running 701 off of the maqetta-test server. On my system, I had this problem, but it went away when I recloned from scratch. Could it be that somehow or other we are building with a non-final version of Dojo?

**Comment by aerwin** [comment]
Just a bit ago, @JonFerraiolo sent me a link to a Dojo bug report found by @peller that seems to cover this issue: http://bugs.dojotoolkit.org/ticket/16085

The Dojo bug report describes it as Chrome-specific, and indeed when I try @JonFerraiolo 's file in FF it works just fine.

I'm running locally from Eclipse after the latest git pull and see the issue in Chrome, so per @JonFerraiolo 's question above, I don't think it has anything to do with potentially building against the wrong level of Dojo.

**Comment by peller** [comment]
The report implies that it's a regression in Chrome 22, so the browser version is also relevant.

**Comment by jhpedemonte** [comment]
According to bug report, this has been fixed in Dojo 1.8.2.  Will keep this issue open until we upgrade to Dojo with correct fix.

**Comment by billreed63** [comment]
I just tested this and it is fixed with the latest dojo in git
